nucypher/cli/main.py

```diff
@@ -46,6 +46,9 @@
     SimpleObserver)
 
 
+FEDERATED_ONLY = False
```

Comment:
Well shit man.  I guess that's how it is.

Reply:
Yeah - Unsure where to put this constant - but It's easy access here.